src/gsy_e/models/market/grid_fees/__init__.py

```diff
@@ -48,46 +48,40 @@ def update_forwarded_offer_with_fee(self, source_rate: float, original_rate: flo
         """Add fees for offer's rate when it's forwarded by another market."""
 
     @abstractmethod
-    def update_forwarded_bid_trade_original_info(
-            self, trade_original_info: TradeBidOfferInfo, market_bid: Bid) -> TradeBidOfferInfo:
-        """
-        When a forwarded bid gets matched in a target market, it will also get matched in
-        the source market with adjustments to the TradeBidOfferInfo of this trade.
-        This method deals with duplicating and updating the TradeBidOfferInfo values.
-        Args:
-            trade_original_info: TradeBidOfferInfo instance created in the target market
-            market_bid: the source bid instance that was cleared in a target market.
-
-        Returns: TradeBidOfferInfo
+    def adapt_bid_fees_on_bid_trade(
```

Comment:
I have noticed that in all occurrences in our codebase, we call either `update_forwarded_bid_trade_original_info` with `propagate_original_bid_info_on_offer_trade` or the `update_forwarded_offer_trade_original_info` with `propagate_original_offer_info_on_bid_trade`, in each of the mentioned methods we recreate a TradeBidOfferInfo object.
SO I simplified the interfaces and left the creation of this instance to the caller code.